CraftTweaker2-API/src/main/java/crafttweaker/api/item/IItemStack.java

```diff
@@ -42,6 +42,14 @@
      */
     @ZenGetter("definition")
     IItemDefinition getDefinition();
+
+    /**
+     * Determine if the itemstack contains an itemblock.
+     *
+     * @return Whether or not the item linked to this stack is an itemblock.
+     */
+    @ZenGetter("isItemBlock")
+    boolean isItemBlock();
```

Comment:
I'm not sure if at this stage adding something nondefault to this interface is safe, since other mods may implement it.

@jaredlll08 what do you think?
Should he replace it with something like
```suggestion
    default boolean isItemBlock() {
        CrafttweakerAPI.logError("Class " + this.getClass().getCanonicalName() + " doesn't override IItemStack#isItemBlock() !");
        return false;
    }
```

Reply:
That is a very fair comment. I'm happy to replace with that suggestion.

Reply:
only saw this now, I would make it a default method